resources/builder.inc.sh

```diff
@@ -199,7 +211,7 @@ builder_echo() {
   message="$*"
 
   if [[ "${action}" == "start" ]] && builder_is_running_on_teamcity; then
-    echo -e "##teamcity[blockOpened name='|[${THIS_SCRIPT_IDENTIFIER}|] ${block}']"
+    $echo_target -e "##teamcity[blockOpened name='|[${THIS_SCRIPT_IDENTIFIER}|] ${block}']"
```

Comment:
flow-wise, isn't this always `echo`?

I only see it changing to builder_error_echo later at l.225.

Reply:
yes, I changed it for consistency